src/app/Playground.jsx

```diff
@@ -0,0 +1,260 @@
+// Framework tools
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+// Playground-specific imports
+import Sidebar from 'components/playground/Sidebar';
+import Customizer from 'components/playground/Customizer';
+import Preview from 'components/playground/Preview';
+import Wizard from 'components/playground/Wizard';
+
+import DocumentTitle from 'react-document-title';
+
+import mediaQueries from 'playgroundSettings';
+
+import MdHelpOutline from 'react-icons/lib/md/help-outline';
+
+// Playground CSS
+require('../../css/playground.css');
```

Comment:
Can't css be required?
if you don't want to get a variable returning from the import you can just do
`import '../../css/playground.css';`